packages/components/scroll-area/src/props.ts

```diff
@@ -307,8 +308,8 @@ export type ScrollAreaScrollbarAxisElement = ScrollAreaScrollbarImplElement
 export interface ScrollAreaScrollbarAxisProps extends Omit<ScrollAreaScrollbarImplProps, keyof ScrollAreaScrollbarImplPrivateProps>, ScrollAreaScrollbarAxisPrivateProps { }
 
 export type ScrollAreaScrollbarAxisPrivateProps = {
-  hasThumb: boolean
-  sizes: Sizes
+  hasThumb: Ref<boolean>
+  sizes: Ref<Sizes>
 }
```

Comment:
There's something weird here, we need to send a dont ref Props. only provide used `Ref`

Reply:
Also, `ScrollAreaScrollbarImplPrivateProps` has the same props with `Ref`, should I remove the `Ref` from both `ScrollAreaScrollbarAxisPrivateProps` and `ScrollAreaScrollbarImplPrivateProps`?

Reply:
yes